tests/input/staging/misc-failures.cs

```diff
@@ -5,14 +5,14 @@ void Func()
 line");
 }
 
-x = o.Func(
-    y);
-x = o.Func2(a, b,
-    y);
+variablex = o.Func(
```

Comment:
Why the change to the variable name?

Reply:
This was suggested by Mihai

Reply:
Ok, for purely visual purposes as I understood. Which is fine. Was just worried that the tokeniser actually behaved differently and that was the reason for the change.